src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsbookavideolinkapi/config/FeatureSwitches.kt

```diff
@@ -29,6 +29,4 @@ class FeatureSwitches(private val environment: Environment) {
     }
 }
 
-enum class Feature(val label: String) {
-  SNS_ENABLED("feature.events.sns.enabled"),
-}
+enum class Feature(val label: String)
```

Comment:
Whilst this is showing as empty now, it will be populated with the toggle for probation meeting/appointment type support in a later PR.